static/js/email_cert.js

```diff
@@ -0,0 +1,19 @@
+function log(obj) {
+    response = JSON.stringify(obj);
+    if (response.search("sent")){
+      alert("Email sent successfully")
+    }
+    else {
+      alert("There was an issue please contact administrator")
```

Comment:
Let's add the support email address here, so they know how to contact the admin.